documentation/explainers/price-feed-identifiers.md

```diff
@@ -0,0 +1,77 @@
+# Price feed identifiers
+
+This document explains how price feed identifiers are configured and how they are used.
+
+## What is an identifier
+
+An identifier is the term we use to refer to an underlying asset whose price a derivative tracks. Examples of underlying assets
+are the price of Bitcoin in US dollars, the spot price of gold, or the daily high temperature in New York City.
+
+## Configuration file
+
+Price feed identifiers are configured in [identifiers.json](`https://github.com/UMAprotocol/protocol/blob/master/core/config/identifiers.json`).
+This json file contains a list of entries for each supported identifier, with two bits of configuration information:
+1. `dappConfig`: configuration for the Sponsor dApp.
+2. `uploaderConfig`: configuration for UMA's price feed uploader.
+See the sections below for details on these configurations.
```

Comment:
Do you think it makes sense to show an example of a configuration object with both a dApp config and uploader config here? To make it less distracting/confusing, we could just `...` out all the details that are explained in later sections:
```
{
  "IDENTIFIER1": {
    "dappConfig": {
      ...
    },
    "uploaderConfig": {
      ...
    }
  },
  "IDENTIFIER2": {
    "dappConfig": {
      ...
    },
    "uploaderConfig": {
      ...
    }
  }
}
```

Reply:
Done.